packages/wrangler/src/triggers/deploy.ts

```diff
@@ -241,7 +241,7 @@ export default async function triggersDeploy(
 	}
 
 	if (config.workflows?.length) {
-		logger.warnOnce("Workflows is currently in open beta.");
+		logger.once.warn("Workflows is currently in open beta.");
```

Comment:
do we need to dbl check this change with the Workflows team?

Reply:
No—this is the same behaviour, it's just a change in how the `warnOnce()` function is defined